src/components/ghost/Ghost.stories.md

```diff
@@ -0,0 +1 @@
+`import Ghost from 'box-ui-elements/es/components/Ghost';`
```

Comment:
Should contents from `README.md` moved over here? I wasn't sure whether to keep or remove the file.

Reply:
I'd recommend moving the opening comment ("The `Ghost` component can be used to create loading states...") here, and then creating stories for the code examples (circle, rectangle, pill, media layout) in `Ghost.stories.tsx`.